api/pkg/apis/v1alpha1/providers/config/catalog/catalogprovider.go

```diff
@@ -188,6 +195,11 @@ func (m *CatalogConfigProvider) ReadObject(ctx context.Context, object string, l
 }
 
 func (m *CatalogConfigProvider) traceValue(v interface{}, localcontext interface{}, dependencyList map[string]map[string]bool) (interface{}, error) {
+	logCtx := context.TODO()
```

Comment:
Why initialize a todo() here? I saw you give the ltx.Context value right after that.

Reply:
add context in signature